service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/SendNoticeOfHearingHandlerTest.java

```diff
@@ -238,6 +248,35 @@ void shouldSendNoticeOfHearingToUnrepresentedOthersByPost() {
         verifyNoInteractions(notificationService);
     }
 
+    @Test
+    void shouldNotifyTranslationTeam() {
+        underTest.notifyTranslationTeam(
+            new SendNoticeOfHearing(CASE_DATA, HearingBooking.builder()
+                .noticeOfHearing(NOTICE_OF_HEARING)
+                .startDate(START_DATE)
+                .translationRequirements(TRANSLATION_REQUIREMENTS)
+                .build())
+        );
+
+        verify(translationRequestService).sendRequest(CASE_DATA,
+            Optional.of(TRANSLATION_REQUIREMENTS),
+            NOTICE_OF_HEARING, "Notice of hearing - 1 March 2012");
+    }
+
+    @Test
+    void shouldNotifyNotifyTranslationTeamIfNoLanguageRequirementDefaultsToEmpty() {
```

Comment:
Should this be "shouldNotifyTranslationTeamIfLanguageRequirementDefaultsToEmpty"?